docs/config/configuring-nodes-storage.rst

```diff
@@ -49,5 +49,10 @@ The following pages describe how to configure the backends currently supported b
    openstack-configuration
    aws-configuration
    emc-configuration
+   vmware-configuration
```

Comment:
Not a new issue, but please address at some point. "supported by Flocker" indicates (or, I infer) that we at ClusterHQ, makers of Flocker, support using these backends.